store/reducers/groups.reducer.js

```diff
@@ -122,105 +124,243 @@ export default function groupsReducer(state = initialState, action) {
         ...newState,
         loading: true,
       };
-    case actions.GROUPS_GETALL_SUCCESS:
+    case actions.GROUPS_GETALL_SUCCESS: {
+      let { groups } = action;
+      const { offline } = action;
+      /* eslint-disable */
+      let localGroups = newState.groups.filter(localGroup => isNaN(localGroup.ID));
+      /* eslint-enable */
```

Comment:
Why is eslint disabled here?

Reply:
When i do the commit, eslint show this error:

disciple-tools-mobile-app/store/reducers/groups.reducer.js
  130:64  error  Unexpected use of 'isNaN'  no-restricted-globals

Reply:
As per the airbnb style guide we are using, this should use `Number.isNaN` instead of `isNaN`
https://github.com/airbnb/javascript#standard-library--isnan

Reply:
Ok, i will do the change.